Toolchain/BuildIt.sh

```diff
@@ -275,6 +277,23 @@ pushd "$DIR"
             echo "Not touching cache then."
         else
             mkdir -p Cache/
+            # We *most definitely* don't need debug symbols in the linker/compiler.
+            # This cuts the uncompressed size from 1.2 GiB per Toolchain down to about 250 MiB.
+            pushd "Local/libexec/gcc/i686-pc-serenity/${GCC_VERSION}"
+                for binary in cc1plus lto1; do
+                    echo "Before: $(du -h "${binary}")"
+                    strip "${binary}"
+                    echo "After: $(du -h "${binary}")"
+                done
+                # C is forbidden anyway by the Contribution Guidelines, so we can delete the C compiler:
```

Comment:
What about Libraries/LibCore/puff.c?

Reply:
Argh! Sorry, we should have waited on Travis to complete.

I see there's already efforts going on to convert puff to C++. If that works: Great!

In case that doesn't work or is too much effort, `BuildIt.sh` doesn't need to delete `cc1`, it's also fine if we just trip it instead. In order to do so delete the handful of lines that `rm cc1`, and add it to the loop around `strip`.

I can't make the commits myself right now because I'm on my work laptop, sorry :/

Reply:
Now that puff.c is renamed, it complains about cc1's lack when building boot.S.